alert.go

```diff
@@ -100,7 +100,8 @@ func parseAlertClause(alert string) (*cm15.AlertSpec, error) {
 	if strings.ToLower(tokens[0]) != "if" {
 		return nil, fmt.Errorf("Alert clause misformatted: missing If. Must be of format: '%s'", alertFmt)
 	}
-	metricTokens := strings.Split(tokens[1], ".")
+	metricValueTokens := strings.Split(tokens[1], "/")
+	metricTokens := strings.Split(metricValueTokens[1], ".")
```

Comment:
I don't think this is going to work correctly as it is going to throw away the relevant `File` portion of the metric. Maybe something like the following is what's needed:

```golang
metric := tokens[1]
period := strings.LastIndex(metric, ".")
if period == -1 {
    return nil, fmt.Errorf("Alert <Metric>.<ValueType> misformatted, should be like 'cpu-0/cpu-idle.value'.")
}
...
alertSpec.File = metric[:period]
alertSpec.Variable = metric[period+1:]
if len(alertSpec.File) == 0 || len(alertSpec.Variable) == 0 {
    return nil, fmt.Errorf("Alert <Metric>.<ValueType> misformatted, should be like 'cpu-0/cpu-idle.value'.")
}
```

Reply:
Implemented this suggestion as it makes sense -- we want to use everything after the last `.` as the `Variable~, everything else is the `File` -- which can include `.`